NineChronicles.Headless/GraphTypes/States/Models/World/ClearedStageType.cs

```diff
@@ -0,0 +1,12 @@
+using GraphQL.Types;
+
+namespace NineChronicles.Headless.GraphTypes.States.Models.World;
+
+public class ClearedStageType : ObjectGraphType<(int, int)>
+{
+    public ClearedStageType()
+    {
+        Field<NonNullGraphType<IntGraphType>>("worldId", resolve: context => context.Source.Item1);
+        Field<NonNullGraphType<IntGraphType>>("stageId", resolve: context => context.Source.Item2);
```

Comment:
It will be better to use named tuple

```suggestion
public class ClearedStageType : ObjectGraphType<(int WorldId, int StageId)>
{
    public ClearedStageType()
    {
        Field<NonNullGraphType<IntGraphType>>("worldId", resolve: context => context.Source.WorldId);
        Field<NonNullGraphType<IntGraphType>>("stageId", resolve: context => context.Source.StageId);
```